.github/workflows/code-quality.yaml

```diff
@@ -29,6 +29,17 @@ jobs:
             - run: yarn --immutable
             - run: yarn ci
 
+    coding-style-php:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v4
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.1'
```

Comment:
we could use the php-cs-fixer tool provided by shivammathur also

Reply:
But we won't be able to run it locally, see https://github.com/symfony/ux/pull/617

Reply:
Yep, easier to use the installed version so you don't have to duplicate version constraints. I forgot that I don't include in my packages - the ci fixes automatically.